Source/FORMData.m

```diff
@@ -700,6 +711,12 @@ - (NSArray *)showTargets:(NSArray *)targets {
                 if (section) {
                     [self sectionWithID:target.targetID completion:^(FORMSection *section, NSArray *indexPaths) {
                         if (section) {
+                            for (FORMField *field in section.fields) {
```

Comment:
@3lvis There is code duplication with Line 684. Maybe could be refactored by method or block?